File: javascript/chapter8/user_album.js
while (true) {
	const album = prompt("Album name: ");
	if (album?.toLowerCase() == "quit") window.close();

	const artist = prompt("Artist name: ");
	if (artist?.toLowerCase() == "quit") window.close();

	console.log(makeAlbum(String(artist), String(album)));
	console.log();
}


/**
 * @param {string} artist
 * @param {string} title
 * @returns {Object<string,string>}
 */
function makeAlbum(artist, title) {
	return { artist, title };
}

File: python/chapter8/user_album.py
def make_album(artist: str, title: str):
    album = {"artist": artist, "title": title}
    return album


while True:
    album = input("Album name: ")
    if album.lower() == "quit":
        break

    artist = input("Artist name: ")
    if artist.lower() == "quit":
        break

    print(make_album(artist, album))
    print()
